cs4360/apps/crm/api/src/app/dao/user/user.controller.ts

```diff
@@ -1,24 +1,45 @@
-import { Controller } from '@nestjs/common';
-import {Crud} from "@nestjsx/crud";
+import {Controller, Type} from '@nestjs/common';
+import {Crud, CrudController, CrudOptions, CrudRequest, Override, ParsedBody, ParsedRequest} from "@nestjsx/crud";
 import {UserEntity} from "@crm/nest/entities";
 import {UserService} from "./user.service";
+import {CreateUserRequest, CreateUserResponse} from "@crm/shared";
 
 
 @Crud({
   model: {
     type: UserEntity
   },
   params:{
-    id:{
+    id: {
       field: 'id',
       type: 'uuid',
       primary: true
     }
-  }
+  },
+  query: {
+    join: {
+      userType: {
+        eager: true, // deep copy down
+      },
+      contact: {
+        eager: true,
+        allow: ['id', 'firstName', 'lastName']
+      }
+    },
```

Comment:
This is how we get nested responses to a GET

Reply:
awesome, looks easy enough for future customization

Reply:
👍